src/main/groovy/grails/plugins/airbrake/AirbrakeAppender.groovy

```diff
@@ -1,28 +1,27 @@
 package grails.plugins.airbrake
 
 import ch.qos.logback.core.AppenderBase
-import ch.qos.logback.classic.Level
-import ch.qos.logback.classic.filter.ThresholdFilter
+import ch.qos.logback.core.joran.spi.ConsoleTarget
+import ch.qos.logback.core.util.EnvUtil
+import ch.qos.logback.core.util.OptionHelper
+
 
 class AirbrakeAppender<E> extends AppenderBase<E> {
 
     private boolean includeEventsWithoutExceptions
+    private final AirbrakeNotifier notifier
 
     AirbrakeAppender(AirbrakeNotifier notifier, includeEventsWithoutExceptions) {
         this.notifier = notifier
         this.includeEventsWithoutExceptions = includeEventsWithoutExceptions
-        // Only notify on events that are Error Level or greater
-        def errorFilter = new ThresholdFilter(level: Level.ERROR)
-        errorFilter.start()
-        this.addFilter errorFilter
```

Comment:
This did not seem to have any effect. 
In doWithApplicationContext of AirbrakeGrailsPlugin we assign the root logger context to the appender so it uses its filter config (defined in logback.groovy). Does that make sense?